src/Altinn.Platform/Altinn.Platform.Events/Events/Migration/v0.03/01-setup-tables.sql

```diff
@@ -0,0 +1,41 @@
+-- Table: events.eventssubscription
+
+CREATE TABLE IF NOT EXISTS events.eventssubscription
+(
+    id BIGSERIAL,
+    sourcefilter character varying COLLATE pg_catalog."default",
+    subjectfilter character varying COLLATE pg_catalog."default",
+    typefilter character varying COLLATE pg_catalog."default",
+    consumer character varying COLLATE pg_catalog."default" NOT NULL,
+    endpointurl character varying COLLATE pg_catalog."default" NOT NULL,
+    createdby character varying COLLATE pg_catalog."default" NOT NULL,
+    validated BOOLEAN NOT NULL,
+    "time" timestamptz  NOT NULL,
+    CONSTRAINT eventssubscription_pkey PRIMARY KEY (id)
+)
+
+
+TABLESPACE pg_default;
+
+CREATE OR REPLACE PROCEDURE events.insert_eventssubcsription(
```

Comment:
I would like to see a split of the scripts in this version so it is easier to go back and find the scripts in the code. Could we set up the procedure in a new file 02-setup-procedure.sql ?

Reply:
I can see we didn't do it for the first versions where the events table was created, so I might have changed my mind..

Reply:
We have functions getsubscription and deletesubscription.. Can we call this insert_subscription?

Reply:
@tba76 fixed that